assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/RequestAccessButton.js

```diff
@@ -29,13 +29,18 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import Link from '../../../../../../components/Link';
+import { Button } from 'googlesitekit-components';
 
 export default function RequestAccessButton( { requestAccessURL } ) {
 	return (
-		<Link danger href={ requestAccessURL } target="_blank">
+		<Button
+			className="googlesitekit-audience-selection-panel__error-notice-action"
```

Comment:
@nfmohit, can we use the `.mdc-button--tertiary` class instead of introducing a new one?

Reply:
Hi Hussain 👋 

As discussed on our call, I did try using the `.mdc-button--tertiary` class (and even `.mdc-button`) but got the Stylelint error `no-descending-specificity` error, which is a false positive in this case. I tried moving the position around but it kept causing issues. I think the ideal solution would be having individual stylesheets for each component, but for now, I've got around it by adding the class name.

Let me know if that sounds good to you.